sdk/communication/communication-chat/src/models/models.ts

```diff
@@ -24,6 +23,23 @@ export interface ChatThreadProperties {
   readonly createdBy?: CommunicationIdentifierKind;
   /** The timestamp when the chat thread was deleted. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. */
   deletedOn?: Date;
+  /** metadata */
+  metadata?: Record<string, string>;
+}
+
+/** Summary information of a chat thread. */
+export interface ChatThreadItem {
+  /** Chat thread id. */
+  id: string;
```

Comment:
Why is this being added (ChatThreadItem)? It's an existing API, right?

Reply:
I reverted that. It doesn't make any API change, was trying to use the manual model instead of generated one.